lib/elixir/lib/map_set.ex

```diff
@@ -347,6 +347,8 @@ defmodule MapSet do
 
   """
   @spec union(t(val1), t(val2)) :: t(val1 | val2) when val1: value, val2: value
+  def union(map_set1, map_set2)
```

Comment:
I was looking at the docs for 1.4.4 but this changed apparently so you're right :)